app/views/accounts/_objective.html.slim

```diff
@@ -1,17 +1,50 @@
-tr
-  td= objective.name
+/ desktop
+tr.for-desktop style="height:90px;"
+  td.is-vcentered style="width: 48px"
+    figure.image.is-48x48
+      img src=asset_path("wizard_goal.svg")
+  td.is-vcentered style="width: 150px"
+    p.span.is-black-budgeting-kid-color
+      = objective.name
+    p.span.has-text-light-gray-budgeting-kid-color
+      = objective.amount
   td
-    = objective.amount
-    br
-    = "it takes #{([objective.amount, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).ceil} weeks"
-  td.has-text-centered
-    =
-    | #{goal_percentage(account, objective)}
-    progress.progress.is-small(value="#{goal_percentage(account, objective).chomp('%')}" max="100") #{goal_percentage(account, objective)}
-  - weeks_to_achieve = objective.weeks_to_achieve
-  td= (weeks_to_achieve == -1 ? "Achieved" : weeks_to_achieve)
+    .columns.mb-1
+      .column.pb-0.is-pulled-left
+        p.pt-2.is-black-budgeting-kid-color
+          = "#{([objective.amount, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).ceil} Weeks to go"
+      .column.pb-0.is-pulled-right
+        = link_to objective_path(objective), method: :delete, data: {confirm: 'Delete this objective?'}, class: 'button is-white has-text-weight-bold has-text-budgeting-kid-color' do
+          figure.image.is-16x16.is-budgeting-kid-light-color
+            = image_tag('delete.svg', alt: '+')
+          | &nbsp; Delete
+    - index = objective_counter + 1
+    progress.progress class="is-small #{index.odd? ? 'is-blue' : 'is-orange'}" style="height:8px;" value="#{goal_percentage(account, objective).chomp('%')}" max="100" #{goal_percentage(account, objective)}
+  td.p-0.is-vcentered style="width: 50px"
+    .title.is-6.pt-4.is-black-budgeting-kid-color
+      | #{goal_percentage(account, objective)}
+
+/ mobile
+tr.for-mobile
+  td.px-0.has-text-centered style="width: 100px;"
+    .columns.p-1.mb-0.is-flex.is-centered
+      .figure.image.is-48x48.m-2
+        img src=asset_path("wizard_goal.svg")
+    p.span.is-black-budgeting-kid-color
+      = objective.name
+    p.span.has-text-light-gray-budgeting-kid-color
+      = objective.amount
   td
-    = link_to objective_path(objective), method: :delete, data: {confirm: 'Delete this objective?'}, class: 'button is-small is-light' do
-      span.icon
-        i.fa.fa-trash-o 
-      | &nbsp; Delete
+    .columns
+      .column.pb-0.is-pulled-left
+        p.is-black-budgeting-kid-color.pt-2
+          = "#{([objective.amount, 0].max / account.automatic_topup_configs.sum(:amount).+(0.1)).ceil} Weeks to go"
```

Comment:
Maybe we may move this into a separate method, like a objective.weeks_to_achieve method?